news/2024-11-22-new-images.md

```diff
@@ -1,7 +1,11 @@
 # Updating our default docker images
 
-TL;DR: We've updated our default docker images to be based on alma9, and we've
-consolidated our image names to follow a consistent pattern:
+TL;DR: We have made some updates to our Docker images and build time GLIBC selection. 
+
+1. We've updated our default docker images to be based on alma9
+2. Our default GLIBC version during builds remains 2.17 (ensuring broad compatibility).
+3. When necessary, it is now possible to build against a newer GLIBC.
```

Comment:
These two haven't really changed? The only change there was decoupling stdlib from the docker_image (for non-cuda recipes, this worked hassle-free already before).

Here's a suggestion
```suggestion
2. It is now easier to override `c_stdlib_version` (especially for CUDA-enabled feedstock), though our baseline of 2.17 hasn't changed
3. Where necessary, you can more easily switch images by setting `os_version: ...` in `conda-forge.yml`
```